### PR TITLE
Adds layer tests for binary and reduce tflite operations

### PR DESCRIPTION
**Pull request by rnugmanx**

### CODE REVIEW

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Binary.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        assert len(set(params.keys()).intersection({'op_name', 'op_func', 'shape'})) == 3, \
+            'Unexpected parameters for test: ' + ','.join(params.keys())
+        self.allowed_ops = [params['op_name']]
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            place_holder0 = tf.compat.v1.placeholder(params.get('dtype', tf.float32), params['shape'],
+                                                     name=TestTFLiteBinaryLayerTest.inputs[0])
+            place_holder1 = tf.compat.v1.placeholder(params.get('dtype', tf.float32), params['shape'],
+                                                     name=TestTFLiteBinaryLayerTest.inputs[1])
+            params['op_func'](place_holder0, place_holder1, name=TestTFLiteBinaryLayerTest.outputs[0])
```
Comment (jane-intel) [inline tests/layer_tests/tensorflow_lite_tests/test_tfl_Binary.py:55]: please also add a test for axis attribute -- None and -1 for example

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py [hunk 2]
```diff
@@ -82,14 +89,14 @@ def _prepare_input(self, inputs_dict, generator=None):
...
     def make_model(self, params):
         assert len(set(params.keys()).intersection({'op_name', 'op_func', 'shape'})) == 3, \
             'Unexpected parameters for test: ' + ','.join(params.keys())
         self.allowed_ops = [params['op_name']]
         tf.compat.v1.reset_default_graph()
         with tf.compat.v1.Session() as sess:
-            place_holder = tf.compat.v1.placeholder(params.get('dtype', tf.float32), params['shape'], name=TestTFLiteUnaryLayerTest.inputs[0])
+            place_holder = tf.compat.v1.placeholder(params.get('dtype', tf.float32), params['shape'],
+                                                    name=TestTFLiteUnaryLayerTest.inputs[0])
             params['op_func'](place_holder, name=TestTFLiteUnaryLayerTest.outputs[0])
```
Comment (jane-intel) [inline tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py:100]: please also add a test for axis attribute -- None and -1 for example

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Reduce.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    def make_model(self, params):
+        assert len(set(params.keys()).intersection({'op_name', 'op_func', 'shape'})) == 3, \
+            'Unexpected parameters for test: ' + ','.join(params.keys())
+        self.allowed_ops = [params['op_name']]
+        tf.compat.v1.reset_default_graph()
+        with tf.compat.v1.Session() as sess:
+            place_holder = tf.compat.v1.placeholder(params.get('dtype', tf.float32), params['shape'],
+                                                    name=TestTFLiteReduceLayerTest.inputs[0])
+            params['op_func'](place_holder, name=TestTFLiteReduceLayerTest.outputs[0])
```
Comment (jane-intel) [inline tests/layer_tests/tensorflow_lite_tests/test_tfl_Reduce.py:47]: please also add a test for axis attribute -- None and -1 for example

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py [hunk 4]
```diff
@@ -17,6 +17,12 @@ def make_positive_array(inputs_dict):
     return inputs_dict
 
 
+def short_range(inputs_dict):
```
Comment (jane-intel) [inline tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py:20]: please take these functions to utils

File: tests/layer_tests/tensorflow_lite_tests/test_tfl_BinaryWithActivation.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    {'shape': [2, 10]}
+]
+
+activations = [
+    {'activation': None},
+    {'activation': tf.nn.relu},
+    {'activation': tf.nn.relu6},
+    {'activation': tf.math.tanh},
+    {'activation': tf.experimental.numpy.signbit},
+    # n1
```
Comment (jane-intel) [inline tests/layer_tests/tensorflow_lite_tests/test_tfl_BinaryWithActivation.py:27]: will you cover the relu n1 case?
Additionally, fused activations are shared between several operations -- you can move this list and activation_helper to some utils.py

Comment (rnugmanx) [reply tests/layer_tests/tensorflow_lite_tests/test_tfl_BinaryWithActivation.py:27]: "will you cover the relu n1 case?" - Yes
"you can move this list and activation_helper to some utils.py" - I will move all such functions in some utils.py

File: tests/layer_tests/common/utils/tflite_utils.py [hunk 6]
```diff
@@ -1,9 +1,60 @@
...
+data_generators = {
+    'positive': make_positive_array,
+    'short_range': short_range,
+    'boolean': make_boolean_array,
+}
+
+
+def activation_helper(input_node, activation_name, name):
+    if activation_name is None:
+        return
```
Comment (jane-intel) [inline tests/layer_tests/common/utils/tflite_utils.py:35]:
```suggestion
        return input_node
```

**Review by jane-intel — APPROVED** [review]
